sql-plugin/src/main/311+-all/scala/com/nvidia/spark/rapids/shims/v2/GpuOrcScan.scala

```diff
@@ -63,7 +63,7 @@ case class GpuOrcScan(
     super.description() + ", PushedFilters: " + seqToString(pushedFilters)
   }
 
-  override def withFilters(
+  def withFilters(
```

Comment:
so I think this is bad, meaning Spark changed this in SPARK-36351 and isn't calling withFilters anymore in 3.3.0.  This implies we might not be getting the filters applied in Spark 3.3.0 because we don't implement the new api.
We need to at least file a follow on for this and point to both SPARK-36351 and this change to investigate to make sure we don't need to use those new API.